acvm-repo/acir_field/src/field_element.rs

```diff
@@ -274,15 +274,19 @@ impl<F: PrimeField> AcirField for FieldElement<F> {
     }
 
     fn to_be_bytes(self) -> Vec<u8> {
-        // to_be_bytes! uses little endian which is why we reverse the output
-        // TODO: Add a little endian equivalent, so the caller can use whichever one
-        // TODO they desire
         let mut bytes = Vec::new();
         self.0.serialize_uncompressed(&mut bytes).unwrap();
         bytes.reverse();
         bytes
     }
 
+    /// Converts the field element to a vector of bytes in little-endian order
+    fn to_le_bytes(self) -> Vec<u8> {
```

Comment:
Unfortunately this will not compile because `to_le_bytes` is not a method on the `AcirField` trait, which is being implemented here.